lib/v1/index.js

```diff
@@ -4,11 +4,14 @@ const { Crypto, Http, Forms } = require('./core');
 const extractDomainFromUrl = require('./utils/domain');
 
 class EvervaultClient {
-  constructor(teamId) {
+  constructor(teamId, apiUrlOverride) {
     if (!Datatypes.isString(teamId)) {
       throw new errors.InitializationError('teamId must be a string');
     }
-    this.config = Config(teamId);
+    if (!Datatypes.isUndefined(apiUrlOverride) || !Datatypes.isString(apiUrlOverride)) {
+      throw new errors.InitializationError('appUrl must be a string');
+    }
```

Comment:
This is an optional argument, we don't want this to be set in general so shouldn't be erroring.